src/v2/view-builder/views/captcha/CaptchaView.js

```diff
@@ -23,6 +23,10 @@ const HCAPTCHA_URL =
 const RECAPTCHAV2_URL = 
   `https://www.google.com/recaptcha/api.js?onload=${OktaSignInWidgetOnCaptchaLoadedCallback}&render=explicit`;
 
+const getCaptchaUrl = (baseURL, locale) => {
```

Comment:
May need to doc that not all of the Widget's supported languages map properly.

For example, our code for Norwegian is `nb`, while [Google's is](https://developers.google.com/recaptcha/docs/language) `no`.

There appears to be one other instance, so we may need to manage a remapping object:
- `nb` 👉🏼  `no`
- `nl-NL` 👉🏼 `nl`

Reply:
@jmelberg-okta I actually noticed this as well when I was reading their docs but thankfully when I tested those scenarios, the mapping worked as you have above, both for reCaptcha and hCaptcha. I think they already perform this mapping on their side.